### DISCUSSION TITLE
Version tag change

### DISCUSSION THREAD

**Discussion by tze-dev** [junegunn/fzf#3916] [General]
Just noticed that the tag of the latest release, `0.54.0`, has different convention to previous releases. It now has a `v` prefix, whereas previous versions don't.

Just want to confirm if this is the intention and is going to be the convention going forward?

This has impact to project like, https://github.com/devops-works/binenv, which currently download binary based on previous naming convention.

**Comment by junegunn** [comment]
Yes, it is noted in the CHANGELOG.

See https://github.com/junegunn/fzf/commit/9e92b6f11e0f59272de410f56493893334071e6e
